pkg/plugins/golang/v3/commons.go

```diff
@@ -79,7 +79,7 @@ manifests: controller-gen`
 		}
 
 		if err := util.ReplaceInFile("Makefile",
-			"ENVTEST_K8S_VERSION = 1.26.1",
+			"ENVTEST_K8S_VERSION = 1.27.1",
```

Comment:
/hold

We should ONLY change it for go/v4 and not for go/v3

Reply:
Ah, got it! Thanks! 
I reverted it just now. :)